lib/classes/PluginManager.js

```diff
@@ -32,7 +32,11 @@ class PluginManager {
   validateCommands(commandsArray) {
     // TODO: implement an option to get deeper than one level
     if (!this.commands[commandsArray[0]]) {
-      throw new this.serverless.classes.Error(`command "${commandsArray[0]}" not found.`);
+      const errorMessage = [
+        `command "${commandsArray[0]}" not found`,
+        ' Run "sls help" for a list of all available commands.',
```

Comment:
`sls` or `serverless`? I like `serverless` more as `sls` might confuse the user if he's not used to the shortcut.